checkov/policies3d/output.py

```diff
@@ -0,0 +1,131 @@
+from __future__ import annotations
+
+import itertools
+import logging
+from collections import defaultdict
+from typing import List, Dict, Any
+
+from colorama import Style
+from prettytable import PrettyTable, SINGLE_BORDER
+from termcolor import colored
+
+from checkov.common.bridgecrew.severities import Severities, BcSeverities
+from checkov.common.output.record import Record, DEFAULT_SEVERITY
+
+
+def compare_cve_severity(cve: Dict[str, str]) -> int:
+    severity = (cve.get("severity") or DEFAULT_SEVERITY).upper()
+    return Severities[severity].level
+
+
+def create_cli_output( *cve_records: list[Record]) -> str:
+    cli_outputs = []
+
+    for record in itertools.chain(*cve_records):
+        cli_outputs.append(record.to_string() + Style.RESET_ALL)
+        if not record.vulnerabilities:
+            #  this shouldn't happen
+            logging.error(f"'vulnerabilities' is not set for {record.check_id}")
+            continue
+
+        package_cves_details_map: dict[str, dict[str, Any]] = defaultdict(dict)
+
+        for cve in record.vulnerabilities:
+            image_name = cve.get('dockerImageName')
+            package_name = cve.get('packageName')
+            package_version = cve.get('packageVersion')
+            severity_str = cve.get('severity', BcSeverities.NONE).lower()
+
+            package_cves_details_map[package_name].setdefault("cves", []).append(
+                {
+                    "id": cve.get('cveId'),
+                    "severity": severity_str
+                }
+            )
+
+            if package_name in package_cves_details_map:
+                package_cves_details_map[package_name]["cves"].sort(key=compare_cve_severity, reverse=True)
+                package_cves_details_map[package_name]["current_version"] = package_version
+                package_cves_details_map[package_name]["image_name"] = image_name
+
+        if len(package_cves_details_map.keys()):
```

Comment:
```suggestion
        if package_cves_details_map:
```
no need to actually calculate the length